node/src/types/block.rs

```diff
@@ -104,6 +106,10 @@ static FINALIZED_BLOCK: Lazy<FinalizedBlock> = Lazy::new(|| {
     let timestamp = *Timestamp::doc_example();
     let secret_key = SecretKey::doc_example();
     let public_key = PublicKey::from(secret_key);
+    let era_report = EraReport::doc_example().clone();
+    let validator_set = era_report.rewards.keys().cloned().collect();
```

Comment:
I just noticed that the `rewards` field is just empty in the `doc_example`... You can leave it as is or change it so that `validator_set` actually contains some keys, either way is fine by me.